src/components/admin/configuration/bulk-operations.tsx

```
@@ -0,0 +1,567 @@
+
+  const handleBulkExport = async () => {
+    setLoading(true);
+    try {
+      let data: any[] = [];
+      
+      switch (entityType) {
+        case 'configuration':
+          data = await ConfigurationService.getAllConfigurations();
+          break;
+        case 'policy':
+          const policyResult = await PolicyService.getAllPolicies(1, 1000);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)